package.json

```diff
@@ -14,7 +14,8 @@
         "lint:fix": "yarn run lint:check --fix",
         "lint": "yarn run lint:check && prettier --check .",
         "postinstall": "patch-package",
-        "prepare": "yarn workspaces run prepare",
+        "prepare": "yarn prepare:husky && yarn workspaces run prepare",
+        "prepare:husky": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
```

Comment:
There are a lot of options for disabling `husky` in CI/CD environments, so I went with this one because it doesn't require any extra dependencies and is pretty simple to setup.

*Reference*: https://typicode.github.io/husky/#/?id=disable-husky-in-cidockerprod

Reply:
Just abstracted it into a file in the `./scripts` directory.